tests/integration/awslambda/test_lambda.py

```diff
@@ -798,6 +800,50 @@ def test_large_payloads(self, caplog, lambda_client, create_lambda_function, sna
         result_data = json.loads(to_str(result_data))
         assert payload == result_data
 
+    @pytest.mark.aws_validated
+    @pytest.mark.skip_snapshot_verify  # skipped - the diff is too big
+    @pytest.mark.skipif(
+        os.environ.get("TEST_TARGET") != "AWS_CLOUD",
```

Comment:
nit: unrelated to the changes in your PR (as we're also using this in other places), but it would be nice if we could encapsulate the `TEST_TARGET` check in a small util function at some point, say, somewhere in `localstack.testing.pytest`, and then re-use it in other places as well. (no need to change this PR, we can tackle this in a follow-up as well 👍 )

Reply:
Good idea, will do so in the next PR!